app/experimenter/nimbus-ui/src/lib/mocks.tsx

```diff
@@ -260,7 +260,16 @@ export const mockExperimentQuery = (
               featureValue: '{"environmental-fact": "really-citizen"}',
               featureEnabled: true,
             },
-            featureConfig: null,
+            featureConfig: {
```

Comment:
@lmorchard, a heads up here that I added a default `featureConfig` to the mock. If this causes headache for you and your branch, I could revert this back to `null` and set a const for a featureConfig mock and pass that in where a non-null feature config is needed.

Reply:
Yeah, I think this will break tests and stories